app/src/main/cpp/Assertions.h

```diff
@@ -0,0 +1,51 @@
+/* -*- Mode: C++; tab-width: 20; indent-tabs-mode: nil; c-basic-offset: 2 -*-
```

Comment:
Why is this here? Just call abort(), there is no need for an assertion macro.

Reply:
Yep. calling abort() is enough for the crash reports, I was wondering if we wanna show some logs for developers before crashes.

If you still think we don't need this, I am happy to remove it.

Reply:
As of right now we don't have a condition where we crash on purpose. I would rather add something when there is a need.